vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj

```diff
@@ -78,7 +78,6 @@
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="ControllableMessageQueueThread.h" />
-    <ClInclude Include="MockExecutorDelegate.h" />
```

Comment:
><ClInclude Include="MockExecutorDelegate.h" /> [](start = 4, length = 46)

Is this intentional? This file shows up as edit instead of removed.

Reply:
Thanks. This was removed at some point during testing but I forgot to readd it.